bin/rucio

```diff
@@ -1801,19 +2464,25 @@ def add_lifetime_exception(args):
                         datasets.append({'scope': scope, 'name': name})
 
     if not datasets:
-        logger.error('Nothing to submit')
+        logger.error('Nothing to submit.')
         return SUCCESS
     try:
         client.add_exception(dids=datasets, account=client.account, pattern='', comments=reason, expires_at=expiration)
     except UnsupportedOperation as err:
         logger.error(err)
         return FAILURE
     except Exception:
-        logger.error('Failure to submit exception. Please retry')
-        logger.debug(traceback.format_exc())
+        if cli_config == 'rich':
+            if logger.level == logging.DEBUG:
+                logger.exception('Failure to submit exception. Please retry.')
```

Comment:
Because this message is the same across all errors, it would make more sense to define the error message as a str var before the logic for picking the logger.